scripts/create-clog.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# create-clog.sh <version> <release-date>
+#
+# Dependencies: sed
+# 
+# Description: Uses template_changelog.md to create a new changelog.
+#
+# Examples:
+#  ./scripts/create-clog.sh "1.18.0" "05/18/2021"
+
+
+set -eou pipefail
+
+VERSION_DELIM="<% X.Y.Z %>"
+RELEASE_DATE_DELIM="<% MM/DD/YYYY %>"
+VERSION=""
+RELEASE_DATE=""
+
+# usage prints information for how to use this program
+usage () {
+  echo "Usage: create-clog <version> <release-date>"
+  echo "Create a changelog from a template"
+  echo "Arugments:"
+  echo "    <version>: A version string 'x.y.z'"
+  echo "    <release-date>: A date string 'mm/dd/yyyy'"
+}
+
+# init parses arguments and initializes all variables.
+init () {
+  lenArgs=2
```

Comment:
consider getopt for this! it's actually not that bad to use, we use it in many of our scripts now...

https://dustymabe.com/2013/05/17/easy-getopt-for-a-bash-script/